junit-platform-suite-commons/src/main/java/org/junit/platform/suite/commons/SuiteLauncherDiscoveryRequestBuilder.java

```diff
@@ -65,7 +65,46 @@
 import org.junit.platform.suite.api.SelectUris;
 
 /**
+ * The {@code SuiteLauncherDiscoveryRequestBuilder} provides a builder for constructing instances
+ * of {@link LauncherDiscoveryRequest} specifically tailored for suite execution.
+ *
+ * <h2>Example</h2>
+ *
+ * <pre>{@code
+ * SuiteLauncherDiscoveryRequestBuilder.request()
+ * 		.selectors(
+ * 			selectPackage("org.example.user"),
+ * 			selectClass("org.example.payment.PaymentTests"),
+ * 			selectClass(File.class),
```

Comment:
`File.class` doesn't make much sense in this context, so I've switched it to `ShippingTests.class` like in `LauncherDiscoveryRequestBuilder`

Reply:
I was testing the code, to make it was correct before adding it to the doc. I used File, just to make sure that it compiles well and I forgot to adapt back to the original 😞

Reply:
Ahhh.... I can totally understand how that happened.

Thanks for actually making sure the examples work!

And don't worry about not having changed it back. 😉